Doc/tutorial/controlflow.rst

```diff
@@ -393,12 +393,15 @@ Several other key features of this statement:
 - Patterns may use named constants.  These must be dotted names
   to prevent them from being interpreted as capture variable::
 
+      import random
       from enum import Enum
       class Color(Enum):
           RED = 0
           GREEN = 1
           BLUE = 2
 
+      color = random.choice(list(Color))
```

Comment:
@ericvsmith to avoid import random, what do you think about this?
```suggestion
      color = Color(int(input("Pick a number between 1 to 3: ")))
```

Reply:
@Fidget-Spinner : It seems okay. But isn't it between 0 and 2?

Reply:
Yeah, typo. Thanks for catching that!

Reply:
Alternative suggestion: why don't we make the values of the `Color` enum strings? Then we don't have to pass the result of `input()` to the `int` constructor before passing it to the `Color` constructor? E.g.:

```python
from enum import Enum
class Color(Enum):
    RED = 'red'
    GREEN = 'green'
    BLUE = 'blue'

color = Color(input("Enter your choice of 'red', 'blue' or 'green': "))
```

instead of

```python
from enum import Enum

class Color(Enum):
    RED = 0
    GREEN = 1
    BLUE = 2

color = Color(int(input("Pick a number between 1 to 2: ")))
```

Reply:
Great! Please go ahead. I'd recommend `input("Enter your choice of 'red', 'blue' or 'green': ").lower()` to be a little more resilient against typos, but it's your call if you want the added complexity. I'll merge either of those.

Reply:
I'll do it without the `.lower()` call, I think -- if we were doing input validation properly, I'd want some kind of a loop that wouldn't let the user progress until they'd entered something valid, and _that's_ getting far too complicated ;)